### DISCUSSION TITLE
Where do I find next.js skilled freelancers

### DISCUSSION THREAD

**Discussion by jesperordrup** [vercel/next.js#11902] [Help]
Are there any freelance sites that nextjs devs usually use?

Freelancer.com doesn't seem to be the place.

**Comment by ravikumarpundr** [comment]
> Are there any freelance sites that nextjs devs usually use?
> 
> Freelancer.com doesn't seem to be the place.

I am a nextjs, react and nodejs full time freelancer. Send me an email at ravi@priorcoder.com if you still looking for someone
